### PR TITLE
Updates UI

### PR DESCRIPTION
**Pull request by philipplentzen**

### CODE REVIEW

**Comment by philipplentzen** [comment]
Draft, because of local storage.

**Comment by philipplentzen** [comment]
@schmetzyannick Take a look 🚀

**Comment by schmetzyannick** [comment]
> Draft, because of local storage.

You can remove the draft state. Local storage in main branch is fine for me.

File: frontend/src/app/app.component.ts [hunk 1]
```diff
@@ -71,6 +84,7 @@ export class AppComponent {
   }
 
   public drop(event: CdkDragDrop<[string, string, boolean][]>) {
+    console.log(event)
```
Comment (schmetzyannick) [inline frontend/src/app/app.component.ts:87]: Remove this

File: frontend/src/app/app.component.scss [hunk 2]
```diff
@@ -1,83 +1,103 @@
+@import "src/utils/variables";
+
+$cards: now, delegate, own, trash;
+$sides: done, backlog;
+
+// Colors
```
Comment (schmetzyannick) [inline frontend/src/app/app.component.scss:6]: Can you move all colors to the variables.scss

**Comment by schmetzyannick** [comment]
Scrollbar should only appear if necessary (in sections/cards)

File: frontend/src/app/app.component.scss [hunk 3]
```diff
@@ -1,83 +1,103 @@
...
+  @media (max-width: 800px) {
+    grid-template-columns: repeat(1, minmax(0, 1fr));
+    grid-template-rows: repeat(6, minmax(0, 1fr));
+    grid-template-areas:
+      "now"
+      "own"
+      "delegate"
+      "trash"
+      "backlog"
+      "done";
```
Comment (schmetzyannick) [inline frontend/src/app/app.component.scss:46]: Should we may not display this column on small windows / screen sizes? It is not a really helpful one.

File: frontend/src/app/app.component.scss [hunk 4]
```diff
@@ -1,83 +1,103 @@
...
+  }
 
-.taskForm{
+  .card {
+    padding: $spacing;
     display: flex;
     flex-direction: column;
-    position: relative;
-    overflow: auto;
-    scrollbar-width: thin;
```
Comment (schmetzyannick) [inline frontend/src/app/app.component.scss:35]: check firefox scrollbar pls

File: frontend/src/app/app.component.scss [hunk 5]
```diff
@@ -1,83 +1,103 @@
...
+      }
+
+      button {
+        background-color: $button;
+      }
+    }
+
+    .taskForm {
+      flex-grow: 1;
+      overflow-y: auto;
```
Comment (schmetzyannick) [inline frontend/src/app/app.component.scss:96]: Scrollbar should only appear if necessary (in sections/cards):
image

**Review by schmetzyannick — COMMENTED** [review]
Review done

File: frontend/src/app/app.component.scss (hunk 3 above)
Comment (philipplentzen) [reply frontend/src/app/app.component.scss:46]: You mean the done "card"?

Comment (schmetzyannick) [reply frontend/src/app/app.component.scss:39]: Yes

Comment (philipplentzen) [reply frontend/src/app/app.component.scss:39]: On mobile it is at the last card. Each card gets 1/3 of the Screenheight, so to see "trash" "backlog" and "done" you need to scroll down. I think its the best way, the important Cards are in the viewport for the Rest just scroll

Comment (schmetzyannick) [reply frontend/src/app/app.component.scss:39]: New Issue

**Pull request merged**